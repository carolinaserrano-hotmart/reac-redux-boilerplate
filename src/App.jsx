import React from 'react';
import { hot } from 'react-hot-loader';

class App extends React.Component {
    render() {
        return (
            <div style={{ display: 'flex', justifyContent: 'center' }}>
                <h1>Hello, there</h1>
            </div>
        );
    }
}

export default hot(module)(App);
