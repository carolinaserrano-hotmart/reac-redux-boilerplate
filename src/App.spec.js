import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import App from './App.jsx';

describe('App', () => {
    it('should shallow render without exploding', () => {
        const wrapper = shallow(<App />);

        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
