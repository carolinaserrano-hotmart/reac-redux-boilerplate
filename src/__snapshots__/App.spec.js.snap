// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should shallow render without exploding 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "justifyContent": "center",
    }
  }
>
  <h1>
    Hello, there
  </h1>
</div>
`;
